.github/workflows/docs.yaml

```diff
@@ -0,0 +1,61 @@
+name: Documentation
+# FOLLOWING https://coderefinery.github.io/documentation/gh_workflow/
+
+on:
+  workflow_dispatch:
```

Comment:
depends on if we want it as an easy way to test in the future, wdyt? it's only allowed by people who have write-access to the repo afaik so it could come in handy